docusaurus/docs/iOS/common-content/reference-docs/stream-chat/controllers/channel-controller/chat-channel-controller-properties.md

```diff
@@ -391,7 +391,7 @@ func createNewMessage(
 
   - text: Text of the message.
   - pinning: Pins the new message. `nil` if should not be pinned.
-  - isSilent: A flag indicating whether the message is a silent message. Silent messages are special messages that don't increase the unread messages count nor mark a channel as unread.
+  - `isSilent`: A flag indicating whether the message is a silent message. Silent messages are special messages that don't increase the unread messages count nor mark a channel as unread.
```

Comment:
Why only this property changed with ``? This is actually creating inconsistency now. Maybe we should disable this rule? Does not seem to work very well.

Reply:
Missed it I thing, we should define a rule of our own that any bulleted item followed by a colon needs backticks. Still learning the tool here. But as you mentioned, API docs will not be in these files in the near future.

The backticks prevent spell checking from flagging these words.